src/anniversary.html

```diff
@@ -1,221 +0,0 @@
-<!DOCTYPE html>
```

Comment:
We might keep this html only for SEO reasons... As react generates the HTML on the fly the SEO for this will be awful. 
But we can improve it a bit if we keep this html. Even if we are not using it in the project. 
On the other hand, I don't know if we will remember to keep it updated... or even if it is worth.
What do you think @simaob ???